test/Business_Logic/Usds/usds.attack.sol

```diff
@@ -0,0 +1,100 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.17;
+
+import "forge-std/Test.sol";
+import {TestHarness} from "../../TestHarness.sol";
+import {TokenBalanceTracker} from '../../modules/TokenBalanceTracker.sol';
+import {IERC20} from "../../interfaces/IERC20.sol";
+import {IWETH9} from '../../interfaces/IWETH9.sol';
+
+contract Exploit_Usds is TestHarness, TokenBalanceTracker {
+    IERC20 internal usds = IERC20(0xD74f5255D557944cf7Dd0E45FF521520002D5748); // sperax token
+    address internal usdsWhale = 0x50450351517117Cb58189edBa6bbaD6284D45902;
+
+    ContractFactory internal factory;
+
+    function setUp() external {
+        cheat.createSelectFork("arbitrum", 57803396); 
+        // deal(address(usds), address(this), 20e18); // because USDS is behind a proxy, this fails on foundry.
```

Comment:
OK, fails, what's the alternative? Or is it not needed here?

Reply:
The alternative, we use the whale pranking method starting from line 21.

Reply:
Should be better explained that that the prank is needed because of this.